.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
     "out": true // set this to false to include "out" folder in search results
   },
   // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-  "typescript.tsc.autoDetect": "off"
+  "typescript.tsc.autoDetect": "off",
+  "prettier.singleQuote": true
```

Comment:
I prefer single quotes rather than double quotes in code, and I didn't find a good way how to deal with it in vscode settings. The `prettier.singleQuote` works as expected for `.js` files but also modifies `.json` files, what I would like to avoid. I also have tried these attrs:

```
"typescript.preferences.quoteStyle": "single",
"javascript.preferences.quoteStyle": "single"
```

none of them work for me.